src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/repository/Node.kt

```diff
@@ -296,6 +298,18 @@ class Node(val repository: Repository, val path: String) : Serializable {
     val json: String
         get() = Formats.toJson(this)
 
+    /**
```

Comment:
new method of repository / node API ;) extremely useful

just `val pkg = repository.node("/content").download()` and we got `File` on disk